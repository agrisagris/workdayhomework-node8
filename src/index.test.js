import ReactDOM from "react-dom";
import React from "react";
import { render } from "@testing-library/react";

test("should render index.js", () => {
  const { container } = render(<div id="root" />);

  const spy = jest.spyOn(ReactDOM, "render");

  require("./index.js");
  expect(spy).toHaveBeenCalled();
  expect(container).toMatchSnapshot();
});
