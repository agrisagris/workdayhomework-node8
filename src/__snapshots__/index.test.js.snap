// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render index.js 1`] = `
<div>
  <div
    id="root"
  >
    <div
      class="container"
    >
      <div>
        Here be dragons
      </div>
      <button
        type="button"
      >
        Load Rates
      </button>
      
    </div>
  </div>
</div>
`;
